scripts/run-artifactory.sh

```diff
@@ -40,3 +43,24 @@ docker cp ${SCRIPT_DIR}/artifactory-1.crt ${CONTAINER_ID_2}:/opt/jfrog/artifacto
 docker cp ${SCRIPT_DIR}/artifactory-2.crt ${CONTAINER_ID_1}:/opt/jfrog/artifactory/var/etc/access/keys/trusted/artifactory-2.crt
 
 echo "Circle-of-Trust is setup between artifactory-1 and artifactory-2 instances"
+
+echo "Generate Admin Access Keys for both instances"
+
+ARTIFACTORY_URLS=("${ARTIFACTORY_URL_1}" "${ARTIFACTORY_URL_2}")
+for ARTIFACTORY_URL in "${ARTIFACTORY_URLS[@]}";
+  do
+    COOKIES=$(curl -c - "${ARTIFACTORY_URL}/ui/api/v1/ui/auth/login?_spring_security_remember_me=false" \
+                  --header "accept: application/json, text/plain, */*" \
+                  --header "content-type: application/json;charset=UTF-8" \
+                  --header "x-requested-with: XMLHttpRequest" \
+                  -d '{"user":"admin","password":"Password1!","type":"login"}' | grep TOKEN)
+
+    REFRESH_TOKEN=$(echo $COOKIES | grep REFRESHTOKEN | awk '{print $7 }')
+    ACCESS_TOKEN=$(echo $COOKIES | grep ACCESSTOKEN | awk '{print $14 }')
```

Comment:
There aren't 14 tokens. I looked at the payload. Both were at position 7. But for portability sake, sed and regex would be more reliable

Reply:
In fact, 14 works as in 2nd row, 7th column. Don't ask me. It just works on both Mac and Ubuntu.

Reply:
I'm still a fan of sed for this. But, you seem to really really like your implementation.

How about extracting a function and then sourcing it?

Reply:
not sure we need this :) but I can do it